apps/mobile/src/libs/types/wheel/sector-parameters-calculation-data.type.ts

```diff
@@ -0,0 +1,11 @@
+type SectorParametersCalculationData = {
```

Comment:
```suggestion
type GetSectorParametersArguments = {
```
If it is only used for getSectorParameters helper, then you can move it directly to that file

Reply:
Use it also in `type WheelAnimationProperties` for `sectorCalculationData` type:
```
type WheelAnimationProperties = {
	animationDuration: number;
	animationRepetitions: number;
	initialSharedValues: SectorInitialSharedValues;
	sectorCalculationData: SectorParametersCalculationData;
};
```